lib/internal/bootstrap/pre_execution.js

```diff
@@ -332,6 +335,60 @@ function initializeDeprecations() {
     enumerable: false,
     configurable: true
   });
+
+  let warnedForProto = false;
+  function warnOnProto() {
+    if (!warnedForProto) {
+      warnedForProto = true;
+      process.emitWarning('usage of Object.prototype.__* properties are a' +
```

Comment:
What if the message here was specialized to tell you the correct method? Since it hits a specific line with `--trace-warnings`, it would make sense.

Reply:
done, though the actual line location is still a bit worthless